components/published-toggle/index.jsx

```diff
@@ -1,13 +1,12 @@
-import React, { useState } from 'react'
+import React from 'react'
 import classNames from 'classnames'
 
 import styles from './index.css'
 
 const PublishedToggle = ({ className, defaultVisibility, isExpanded }) => {
```

Comment:
Could you please provide `disabled` prop here and just disable it from parent?

Disabled toggle should have transparent background and different cursor.

```css
.disabled {
  opacity: 0.5; /* or any other */
  cursor: not-allowed;
}
```